managed-ledger/src/main/proto/MLDataFormats.proto

```diff
@@ -124,3 +124,8 @@ message ManagedCursorInfo {
     // Store which index in the batch message has been deleted
     repeated BatchedEntryDeletionIndexInfo batchedEntryDeletionIndexInfo = 7;
 }
+
+message ManagedLedgerInfoMetadata {
+    required string compressionType = 1;
```

Comment:
Use compression enum type instead of String？

Reply:
The `CompressionType` in the `PulsarApi.proto`, if we want to use the enum `CompressionType`, we need import `PulsarApi.proto`, maybe we could use `string` type here?

Reply:
I think we could just copy the same enum type here, or even declare it as `int` and use the other enum type

Reply:
Looks we'd better to copy the enum type to the MLDataFormats.proto because the managed ledger module can be a independent component.